include/alpaka/mem/buf/uniformCudaHip/Set.hpp

```diff
@@ -94,8 +94,6 @@ namespace alpaka
             template<typename TQueue>
             auto enqueue(TQueue& queue) const -> void
             {
-                using Idx = Idx<TExtent>;
-
```

Comment:
Is this a related change?

Reply:
No, as documented in the PR description and the separate commit.